compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2937,12 +2940,203 @@ OMR::ARM64::TreeEvaluator::arraycmpEvaluator(TR::Node *node, TR::CodeGenerator *
 	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
 	}
 
+static void
+inlineArrayCopy(TR::Node *node, int64_t byteLen, TR::Register *srcReg, TR::Register *dstReg, TR::CodeGenerator *cg)
```

Comment:
Perhaps this would be better named `inlineConstantLengthArrayCopy` (or something to that effect) to reflect that it is generating code to copy a compile-time constant length.

Reply:
I renamed the function as suggested.